### PR TITLE
Fixing security vulnerabilities

### PR DESCRIPTION
**Pull request by aayush3011**
## Type of PR 
Fixing security vulnerabilities
- Updated the azure-cosmos version to 4.41.0
- Updated the kafka-connect-client to 4.0.3
- Added guava as a dependency

## Observability + Testing
- What changes or considerations did you make in relation to observability?

- Did you add testing to account for any new or changed work?

## Issues Closed or Referenced

- Closes #<issue number> (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -130,7 +130,13 @@
         <dependency>
             <groupId>org.sourcelab</groupId>
             <artifactId>kafka-connect-client</artifactId>
-            <version>3.1.0</version>
```
Comment (aayush3011) [inline pom.xml:133]: Updated the kafka-connect-version, as it brings jackson-databind as a dependency. And jackson-databind needed to be updated to fix the security issues.

File: pom.xml [hunk 2]
```diff
@@ -130,7 +130,13 @@
...
+            <version>4.0.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```
Comment (aayush3011) [inline pom.xml:137]: Added guava as a dependency, because kafka-connect-client new version removed it from it's dependency.

**Review by kushagraThapar — DISMISSED** [review]

File: pom.xml (hunk 1 above)
Comment (FabianMeiswinkel) [reply pom.xml:133]: what was the jackson-databind vulnerability? Just the max hierarchy depth default taht could cause denial of service attacks?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## Release History
 
+### 1.7.0
+#### New Features
+* Updated `azure-cosmos` version to 4.41.0.
```
Comment (FabianMeiswinkel) [inline CHANGELOG.md:5]: Please also call out the upgrade of kafka connect dependency

**Review by FabianMeiswinkel — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (aayush3011) [reply CHANGELOG.md:5]: Added

File: pom.xml (hunk 1 above)
Comment (aayush3011) [reply pom.xml:133]: These were the two vulnerabilities
- https://avd.aquasec.com/nvd/2022/cve-2022-42003/
- https://avd.aquasec.com/nvd/2022/cve-2022-42004/

File: CHANGELOG.md (hunk 3 above)
Comment (FabianMeiswinkel) [reply CHANGELOG.md:5]: never mind - just a test dependency - missed that.

File: pom.xml (hunk 1 above)
Comment (FabianMeiswinkel) [reply pom.xml:133]: Yes - thanks

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**